nuget/Microsoft.Windows.CppWinRT.targets

```diff
@@ -112,8 +117,17 @@ Copyright (C) Microsoft Corporation. All rights reserved.
         </ItemGroup>
     </Target>
 
-    <Target Name="CppWinRTResolveWinMD"
-            Condition="'@(CppWinRTMdMergeInputs)' != '' AND '$(CppWinRTEnableComponentProjection)' == 'true'"
+    <Target Name="ComputeCppWinRTResolvedWinMD"
+            Condition="'$(CppWinRTGenerateWindowsMetadata)' == ''"
+            DependsOnTargets="GetCppWinRTMdMergeInputs">
+        <PropertyGroup>
+            <CppWinRTGenerateWindowsMetadata Condition="'@(CppWinRTMdMergeInputs)'!= ''">true</CppWinRTGenerateWindowsMetadata>
```

Comment:
shouldn't conditions also contain '$(CppWinRTGenerateWindowsMetadata)'=='' ?  (i.e., these are overridable defaults)

Reply:
Already checked in the target condition, though we could duplicate it. What do you think?

Reply:
ah, missed that.  in that case, maybe rename the target to clarify that its sole purpose is to assign this property.  also, should targets referencing this property depend on this target?

Reply:
Because they have conditions that won't work correctly. They order they are added to the AfterMidlTargets property will guarantee the correct order.